.storybook/lucid-docs-addon/PanelToggles.js

```diff
@@ -1,34 +1,39 @@
 import React from 'react';
 import Hoverable from './Hoverable';
+import addons, { makeDecorator } from '@storybook/addons';
+import ReactDOM from 'react-dom';
 
-export default class extends React.Component {
-	constructor(...args) {
-		super(...args);
-		this.onToggleLayout = this.onToggleLayout.bind(this);
-		this.onTogglePanel = this.onTogglePanel.bind(this);
-	}
-
-	onToggleLayout() {
-		const urlState = this.props.api.getUrlState();
-		this.props.api.setOptions({
-			addonPanelInRight: !urlState.panelRight,
-		});
-	}
+const portalElement = window.document.createElement('div');
+window.parent.document.body.appendChild(portalElement);
 
-	onTogglePanel() {
-		const urlState = this.props.api.getUrlState();
-		this.props.api.setOptions({
-			showAddonPanel: !urlState.addons,
-		});
-	}
+export const withPanelToggles = makeDecorator({
```

Comment:
Using storybook's decorator functionality since we only want the toggle visible on "example" pages and not our "article" pages.